src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -1,714 +1,734 @@
-/*
```

Comment:
I'm not sure what you did. Perhaps you replaced the LF endings with CRLF? Remove all the `\r` please.  Git thinks you rewrote this entire file.